packages/tools/pie-icons-webc/README.md

```diff
@@ -0,0 +1,113 @@
+
+# pie-icons-webc
+
+Shared PIE Icon Components for Web Components.
+
+This package generates an Web Component iconset for applications using the base [pie-icons](https://www.npmjs.com/package/@justeattakeaway/pie-icons) package. The SVGs in pie-icons are compiled into single file components that can be imported into applications.
+
+---
+
+[![npm version](https://img.shields.io/npm/v/@justeattakeaway/pie-icons-webc.svg)](https://img.shields.io/npm/v/@justeattakeaway/pie-icons-webc.svg)
+
+---
+## Usage
+
+### Installation
+
+Add the module to your project
+
+```bash
+yarn add @justeattakeaway/pie-icons-webc
+```
+
+### Applications
+
+```js
+// Only import what you need!
+import { IconAppRestaurant, IconAppRestaurantLarge } from '@justeattakeaway/pie-icons-webc';
+```
+
+Within the context of a Vue app, that will look like:
+
+  ```js
+  <template>
+      <icon-calendar />
+      <icon-app-restaurant size="xs"/>
+  </template>
+
+  <script>
+  import { IconAlcoholFilled } from '@justeattakeaway/pie-icons-webc';
```

Comment:
I think this should be: `{ IconCalendar, IconAppRestaurant }`

Reply:
whoops thank you!